### PR TITLE
VACMS-3448: add test for revision_log field

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #3448 

## Testing done

Ran new test, visually confirmed fix on incorrect node types.

## Screenshots
Before:
Screen Shot 2020-11-10 at 1 57 42 PM

After:
Screen Shot 2020-11-10 at 1 58 05 PM

## QA steps

As a PM familiar with the content requirements:

1. Review the list of modified content types:
- [ ] All modified content types should have the revision_log field at the bottom of the node edit form

## Definition of Done
- [ ] Documentation has been updated, if applicable.

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by acrollet** [comment]
@oksana-c if you don't mind reviewing the changes I'd appreciate it, thanks!

@ElijahLynn another new test to look at...

**Comment by acrollet** [comment]
Also @oksana-c I assume no announcement is needed but asking about it since it does change the editor UI on the affected content types.

**Review by oksana-c — DISMISSED** [review]
thanks @acrollet ! 
the addition of a dedicated test is great 
Screen Shot 2020-11-12 at 6 57 45 AM

no announcement is required. Missing revision log field was a regression from a previous state.

File: scripts/tests/check-revision-logs.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (ElijahLynn) [inline scripts/tests/check-revision-logs.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/tests/check-revision-logs.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export FAILURE=0
+for filename in config/sync/core.entity_form_display.node.*.default.yml
+do
+  if grep -q -A1 '\- moderation_state' ${filename}
```
Comment (ElijahLynn) [inline scripts/tests/check-revision-logs.sh:6]: Can we use long options here for self-documenting code?
```suggestion
  if grep --quiet --after-context=1 '\- moderation_state' ${filename}
```

File: tests.yml [hunk 3]
```diff
@@ -57,3 +57,7 @@ va/tests/behat:
   command: |
     cd tests/behat
     behat --colors
+
+va/tests/revision-log:
+  description: Ensure revision log field is present
+  command: ./scripts/tests/check-revision-logs.sh
```
Comment (ElijahLynn) [inline tests.yml:63]: I wonder if this should be in the tests folder, since it is the test.

Comment (acrollet) [reply tests.yml:63]: I was going back and forth about that, sounds good.

**Review by ElijahLynn — APPROVED** [review]

**Pull request merged**